src/components/Button/Button.css.ts

```diff
@@ -0,0 +1,170 @@
+import { recipe, RecipeVariants } from "@vanilla-extract/recipes";
+import { sprinkles } from "~/theme";
+
+export const button = recipe({
+  base: [
+    sprinkles({
+      fontFamily: "body",
+      outlineStyle: "none",
+      display: "flex",
+      placeItems: "center",
+      cursor: "auto",
+      padding: 0,
+    }),
+  ],
+
+  variants: {
+    variant: {
+      primary: sprinkles({
+        backgroundColor: {
+          default: "interactiveNeutralDefault",
+          hover: "interactiveNeutralHovering",
+          focusVisible: "interactiveNeutralFocused",
+          active: "interactiveNeutralPressing",
+          disabled: "interactiveNeutralDisabled",
+        },
+        color: {
+          default: "textNeutralContrasted",
+          disabled: "textNeutralDisabled",
+        },
+        borderStyle: "none",
+      }),
+      secondary: sprinkles({
+        backgroundColor: {
+          default: "interactiveNeutralSecondaryDefault",
+          hover: "interactiveNeutralSecondaryHovering",
+          focusVisible: "interactiveNeutralSecondaryFocused",
+          active: "interactiveNeutralSecondaryPressing",
+          disabled: "interactiveNeutralSecondaryDisabled",
+        },
+        color: {
+          default: "textNeutralDefault",
+          disabled: "textNeutralDisabled",
+        },
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderColor: {
+          default: "neutralDefault",
+          hover: "neutralSubdued",
+          focusVisible: "neutralDefault",
+          active: "neutralSubdued",
+          disabled: "neutralPlain",
+        },
+      }),
+      tertiary: sprinkles({
+        backgroundColor: {
+          default: "interactiveNeutralHighlightDefault",
+          hover: "interactiveNeutralHighlightHovering",
+          focusVisible: "interactiveNeutralHighlightFocused",
+          active: "interactiveNeutralHighlightPressing",
+          disabled: "interactiveNeutralHighlightDisabled",
+        },
+        color: {
+          default: "iconNeutralDefault",
+          disabled: "iconNeutralDisabled",
+        },
+        borderStyle: "none",
+      }),
+    },
+    size: {
+      sm: {},
```

Comment:
**suggestion**: Consistency

I propose we use full names here - to be consistent with other parts of MacawUI as `Text` or `Icon`